src/renderer/services/interfaces.ts

```diff
@@ -1,5 +1,31 @@
+import {Keypair} from "@chainsafe/bls/lib/keypair";
+import {bytes} from "@chainsafe/eth2.0-types";
+
 export interface IService {
     start(): Promise<void>;
 
     stop(): Promise<void>;
 }
+
+export interface ICGKeystore {
+    decrypt(password: string): Keypair;
+    changePassword(oldPassword: string, newPassword: string): void;
+    //deletes physical keystore file
+    destroy(): void;
+    create(file: string, password: string, keypair: Keypair): ICGKeystore;
+}
+
+export interface ICGKeystoreFactory {
+    new (file: string): ICGKeystore;
```

Comment:
maybe it would be nicer if this wan't constructor rather "create" method;
You could make if in test to return sino.stub()